EXPLAINER.md

```diff
@@ -58,7 +58,22 @@ Lots of examples can be found here: https://w3c.github.io/web-nfc/#examples
 
 ## Key scenarios
 
-Web NFC works well with other existing APIs
+Among different use-cases, NFC allows users to take their NFC enabled device
+such a a phone and touch NFC tags (sticker, card at the like). The platform,
+or a certain application can then at this moment read the content and write
+new content back.
+
+Web NFC allows to build web experiences that can do the above, thus exchange
+data between the web enabled device and NFC tags.
+
+Data read from NFC tags can come from unknown sources and thus should be
+treated like that by the web developer. This is no different than treating
+user input and data such as JSON, fetched from a remote server.
+
+NDEF signature records can be done to validate that data comes from a trusted
```

Comment:
It's not validation that signature records enable, but content integrity (i.e. no one has tampered with the signed content).

Reply:
Well you can encode id right...

Reply:
How would you rewrite it?

Reply:
It's also authentication, as there is a chain of certificates in a signature.

The Signature RTD spec says 

> It provides users with the possibility of verifying the authenticity and integrity of data within the NDEF message.